chainermn/communicators/_memory_utility.py

```diff
@@ -204,13 +204,16 @@ def _batched_pack_params(params_data, buffer, dtype):
     buf_dtype = _communication_utility._get_nccl_type_id(dtype)
     n_threads = 128
     n_blocks = (n_elems + n_threads - 1) // n_threads
-    _cupy_batched_pack_params()(
-        (n_blocks, ), (n_threads, ),
-        (buffer.memory.ptr, buf_dtype, n_elems,
-         params_dptr, params_dtype, params_size_csum, n_params))
+    if stream is None:
+        stream = cp.cuda.get_current_stream()
```

Comment:
Should we always use stream/async?

Reply:
No.
`cp.cuda.get_current_stream()` return null stream in default case.